security/ssh_certificate.ksy

```diff
@@ -0,0 +1,159 @@
+        doc: |
+          The signature key field contains the CA key used to sign the
+          certificate. The valid key types for CA keys are ssh-rsa,
+          ssh-dss, ssh-ed25519 and the ECDSA types ecdsa-sha2-nistp256,
+          ecdsa-sha2-nistp384, ecdsa-sha2-nistp521. "Chained" certificates, where
+          the signature key type is a certificate type itself are NOT supported.
+          Note that it is possible for a RSA certificate key to be signed by a
+          Ed25519 or ECDSA CA key and vice-versa.
+        type: cstring_sshkey
+      - id: signature
+        doc: |
+          signature is computed over all preceding fields from the initial string
+          up to, and including the signature key. Signatures are computed and
+          encoded according to the rules defined for the CA's public key algorithm
+          (RFC4253 section 6.6 for ssh-rsa and ssh-dss, RFC5656 for the ECDSA
+          types), and draft-josefsson-eddsa-ed25519-03 for Ed25519.
+        type: packed_cstring_tuple
+    enums:
+      cert_type:
+        1: user
+        2: host
+  packed_cstring:
+    seq:
+      - id: len_packed_cstring
+        type: u4
+      - id: packed_strings
+        size: len_packed_cstring
+        type: cstrings
+  packed_cstring_tuple:
+    seq:
+      - id: len_packed_cstring_tuple
+        type: u4
+      - id: packed_strings
+        type: cstring_tuples
+        size: len_packed_cstring_tuple
+  cstrings:
+    seq:
+      - id: strings
+        type: cstring_utf8
+        repeat: eos
+  cstring_tuples:
+    seq:
+      - id: strings
+        type: cstring_tuple
+        repeat: eos
+  cstring_tuple:
+    seq:
+      - id: name
+        type: cstring_utf8
+      - id: data
+        type: cstring_bytes
+  cstring_sshkey:
+    seq:
+      - id: len_cstring_sshkey
+        type: u4
+      - id: value
+        type: ssh_public_key
+  cstring_bytes:
+    doc-ref: 'https://tools.ietf.org/html/rfc4251#section-5'
+    doc: |
+      Arbitrary length binary string. Strings are allowed to contain
+      arbitrary binary data, including null characters and 8-bit
+      characters. They are stored as a uint32 containing its length
+      (number of bytes that follow) and zero (= empty string) or more
+      bytes that are the value of the string. Terminating null
+      characters are not used.
+
+      Strings are also used to store text. In that case, US-ASCII is
+      used for internal names, and ISO-10646 UTF-8 for text that might
+      be displayed to the user. The terminating null character SHOULD
+      NOT normally be stored in the string. For example: the US-ASCII
+      string "testing" is represented as 00 00 00 07 t e s t i n g. The
+      UTF-8 mapping does not alter the encoding of US-ASCII characters.
+    seq:
+      - id: len_cstring_bytes
+        type: u4
+      - id: value
+        size: len_cstring_bytes
+  cstring_utf8:
+    doc: variant of cstring_bytes that decodes to UTF-8
+    seq:
```

Comment:
This improves usability in the [Web IDE](https://ide.kaitai.io/), because you don't have to expand the type to see what's inside:

```suggestion
    -webide-representation: '{value}'
    seq:
```